### ISSUE TITLE
Feature Request - Add Images to templates

### ISSUE THREAD

**Issue by Moneysac90** [Syslifters/sysreptor#56]
We have just seen that it is not possible to add images to templates. Therefore, if your template contains images, you have to add them to the corresponding finding every time. This is contrary to the purpose of a template and makes writing the report a lot more difficult.

**Comment by aronmolnar** [comment]
Yes, we are aware of this and this is definitely on our roadmap.

**Comment by Moneysac90** [comment]
Thanks a lot for your quick response and keep up the great work!

**Comment by MWedl** [comment]
Implemented in https://github.com/Syslifters/sysreptor/releases/tag/0.110

**Comment by aronmolnar** [comment]
![images_in_templates](https://github.com/Syslifters/sysreptor/assets/25219407/0abb9251-7e50-4500-8851-4cb872c33d0e)
